tests/roxctl/roxctl-k8s-context.sh

```diff
@@ -22,7 +22,7 @@ test_roxctl_cmd() {
   kubectl get ep central
 
   # Verify central whoami using current k8s context.
-  if OUTPUT=$(roxctl --insecure-skip-tls-verify -p "$ROX_PASSWORD" central whoami --use-current-k8s-context \
+  if OUTPUT=$(roxctl -p "$ROX_PASSWORD" central whoami --use-current-k8s-context \
```

Comment:
Would be great to have a test case (if we can manage it) with a constrained RBAC user to try the `use-current-k8s-context`.